ui/raidboss/data/07-dt/dungeon/skydeep-cenote.txt

```diff
@@ -0,0 +1,244 @@
+# -p 8E60:2013.1
+
+# Vurgar Mettlegrounds will be sealed off
+2000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "13A8" } window 2000,1
+2008.1 "--sync--" StartsUsing { id: "8E60", source: "Firearms" } window 8,5
+2013.1 "Dynamic Dominance" Ability { id: "8E60", source: "Firearms" }
+2023.3 "Mirror Maneuver" Ability { id: "98E3", source: "Firearms" }
+2025.3 "--north--" Ability { id: "8E63", source: "Firearms" }
+2037.7 "Thunderlight Burst" Ability { id: "8E5D", source: "Firearms" }
+2043.0 "--center--" Ability { id: "8E63", source: "Firearms" }
+2047.3 "Ancient Artillery" Ability { id: "8E5A", source: "Firearms" }
+2055.3 "Emergent Artillery" Ability { id: "9858", source: "Firearms" }
+2060.7 "Artillery" Ability { id: "9704", source: "Firearms" }
+2071.7 "Pummel" Ability { id: "8E5F", source: "Firearms" }
+
+# begin loop
+2081.8 "Mirror Maneuver" Ability { id: "98E3", source: "Firearms" }
+2083.8 "--north--" Ability { id: "8E63", source: "Firearms" }
+2096.4 "Thunderlight Burst" Ability { id: "8E5D", source: "Firearms" }
+2106.7 "Dynamic Dominance" Ability { id: "8E60", source: "Firearms" }
+2108.8 "--center--" Ability { id: "8E63", source: "Firearms" }
+2113.1 "Ancient Artillery" Ability { id: "8E5A", source: "Firearms" }
+2121.1 "Emergent Artillery" Ability { id: "9858", source: "Firearms" }
+2126.5 "Artillery" Ability { id: "9704", source: "Firearms" }
+2129.5 "Thunderlight Flurry" Ability { id: "8E62", source: "Firearms" }
+2141.4 "Mirror Maneuver" Ability { id: "98E3", source: "Firearms" } forcejump 2081.8
+
+# IGNORED ABILITIES
+# 8E5B Thunderlight Burst (boss-targeted)
+# 96B5 Thunderlight Burst (seequential cleave)
+# 96B6 Thunderlight Burst (seequential cleave)
+# 96B7 Thunderlight Burst (seequential cleave)
+# 96B8 Thunderlight Burst (seequential cleave)
+# 9705 Artillery (concurrent with 9704)
+# 9706 Artillery (concurrent with 9704)
+# 9707 Artillery (concurrent with 9704)
+
+# ALL ENCOUNTER ABILITIES
+# 368 attack
+# 8E5A Ancient Artillery
+# 8E5B Thunderlight Burst
+# 8E5D Thunderlight Burst
+# 8E5F Pummel
+# 8E60 Dynamic Dominance
+# 8E62 Thunderlight Flurry
+# 8E63 --sync--
+# 96B5 Thunderlight Burst
+# 96B6 Thunderlight Burst
+# 96B7 Thunderlight Burst
+# 96B8 Thunderlight Burst
+# 9704 Artillery
+# 9705 Artillery
+# 9706 Artillery
+# 9707 Artillery
+# 9858 Emergent Artillery
+# 98E3 Mirror Maneuver
+
+#~~~~~~~~~~~#
+# Maulskull #
+#~~~~~~~~~~~#
+
+# -ii 8F46 8F40 8F43 8F44 8F64 8F42 8F50 8F4E 8F53 8F54 8F3A 98D1 8F60 8F61 8F67 8F51
+# -p 8F3D:3013.1
+
+# Gatekeep's Anvil will be sealed off
+3000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "13A9" } window 3000,1
+3005.2 "--sync--" StartsUsing { id: "8F3D", source: "Maulskull" }
+3013.1 "--sync--" Ability { id: "8F3D", source: "Maulskull" }
+3014.1 "Stonecarver (first)" Ability { id: "8F3E", source: "Maulskull" }
+3016.7 "Stonecarver (second)" #Ability { id: "8F3F", source: "Maulskull" }
+3030.4 "Impact + Skullcrush" Ability { id: "8F45", source: "Maulskull" }
```

Comment:
This is worth labeling as Skullcrush, since it's a visible cast, but I think elsewhere, we might either label it as `Impact + Stonecarver` or simply as `Colossal Impact`. This would better match up with the castbars.

Reply:
> This is worth labeling as Skullcrush, since it's a visible cast, but I think elsewhere, we might either label it as `Impact + Stonecarver` or simply as `Colossal Impact`. This would better match up with the castbars.

The label `Impact + Skullcrush` actually gets used for two different IDs: `8F3B`, which is the center knockback that's followed by two Stonecarvers, and `8F63` which is the front-corner knockback that's followed by a stack/spread.  The first is actually tied to the `Ringing Blows` cast, and the second is tied to `Colossal Impact`.  I'm thinking it might make sense to relabel accordingly (and just remove the existing entries for `Ringing Blows`, since they are castbar-only and the actual damage snapshot/knockback is 1.9s later on the `Impact + Skullcrush`)?

Reply:
> I'm thinking it might make sense to relabel accordingly (and just remove the existing entries for Ringing Blows, since they are castbar-only and the actual damage snapshot/knockback is 1.9s later on the Impact + Skullcrush)?

That seems reasonable, yes. Let's go with that.